usage/app/lib/UsageConfig.scala

```diff
@@ -64,8 +64,8 @@ class UsageConfig(resources: GridConfigResources) extends CommonConfig(resources
     try {
       iamClient.getUser.getUser.getUserName
     } catch {
-      case e:com.amazonaws.AmazonServiceException=>
-        logger.warn("Unable to determine current IAM user, probably because you're using temp credentials.  Usage may not be able to determine the live/preview app names")
+      case e:com.amazonaws.SdkClientException =>
```

Comment:
This is the parent exception to AmazonServiceException - this change captures the client side error as well as server error responses.